### DISCUSSION TITLE
Can the ngrok integration allow me to access additional FQDNs like DDEV can?

### DISCUSSION THREAD

**Discussion by dualhub** [ddev/ddev#5844] [Q&A]
My PHP project has 2 applications running on the same DDEV project:

- Frontend (main website - local.mydomain.com)
- Backend (admin area - localadmin.mydomain.com)

This works well locally using DDEV, but I'd like to be able to share both the frontend and the backend domains using Ngrok. 
Currently I can access just the frontend using Ngrok from outside of my local network, but not the backend.

Does anyone know if this might be possible? 

If you're interested in why we might want to do this, it is so we can temporarily preview a git feature branch to a client.

Thanks in advance for any input.

**Comment by stasadev** [comment]
Hi @dualhub,

Although I haven't tried it myself, I found this answer on how to set up a proxy for the backend.
https://stackoverflow.com/a/58767477/8097891

**Reply by dualhub** [reply]
Gonna check this out - thanks @stasadev

**Comment by rfay** [comment]
Something to know about `ddev share` is that it's a super-simple command. All it does is launch ngrok (and shows you the command it uses.) You could easily create a custom (shell) command for DDEV that launches more than one ngrok or launches with a different configuration. Probably `ddev share` should be turned into a shell command to make it easier for people to alter.  (Also, there are many competitors to ngrok that would work fine this way.)

**Reply by dualhub** [reply]
That's really helpful, Randy - thanks. Would be cool to see that 'share' shell command one day.
